src/microvis/core/view.py

```diff
@@ -138,10 +138,10 @@ def show(self) -> Canvas:
     def add_node(self, node: NodeType) -> NodeType:
         """Add any node to the scene."""
         self.scene.add(node)
-        if self.camera.has_adaptor:
-            # FIXME!: put this vispy specific API elsewhere
-            # i guess we need a reset_range type API
-            self.camera.native.set_range(margin=0)
+        # if self.camera.has_adaptor:
+        #     # FIXME!: put this vispy specific API elsewhere
+        #     # i guess we need a reset_range type API
+        #     self.camera.native.set_range(margin=0)
```

Comment:
this used the native API directly - have commented out for now and tests are still passing but as you say, we need a fix. What does the reset_range API you envision mean? I'm never exactly sure what 'range' refers to, is it some notion of the 'viewing area'?

Reply:
Yeah, the api needs some consideration.  The effect of this call to set_range(margin=0) here is essentially to set the zoom (and position?) such that the image exactly fills the viewport.  Not sure we need a backend call for this at all actually, it seems it should be possible with zoom and center position alone right?  (This was just a short term patch to get me something to look at in the canvas)

This also reminds me, our zoom and position docstring/definitions should be explicit about the units/spaces in which they are defined.  (Ie, what is a zoom of 1)